### PR TITLE
Add docs for `HTMLIFrameElement.allow`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

spec at https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-allow

### Related issues and pull requests

part of the https://github.com/mdn/mdn/issues/520

### CODE REVIEW

File: files/en-us/web/api/htmliframeelement/allow/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+The **`allow`** property of the {{domxref("HTMLIFrameElement")}} interface indicates the [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy) specified for this `<iframe>`. The policy defines what features are available to the `<iframe>` (for example, access to the microphone, camera, battery, web-share, etc.) based on the origin of the request.
+
+The Permissions Policy specified by the `allow` attribute implements a further restriction on top of the policy specified in the {{HTTPHeader("Permissions-Policy")}} header. It doesn't replace it.
+
+It reflects the `allow` attribute of the {{HTMLElement("iframe")}} element.
+
+## Value
+
+A string.
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/allow/index.md:19]: We must inform the reader here that this is a space-separated list of policies. Also, we should be sure the list of potential entries is one click away. Is there an explanation under `<iframe>` we could link to?

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/htmliframeelement/allow/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+slug: Web/API/HTMLIFrameElement/allow
+page-type: web-api-instance-property
+browser-compat: api.HTMLIFrameElement.allow
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`allow`** property of the {{domxref("HTMLIFrameElement")}} interface indicates the [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy) specified for this `<iframe>` element. The policy defines what features are available to the `<iframe>` element (for example, access to the `microphone`, `camera`, `battery`, `web-share`, etc.) based on the origin of the request.
+
+The Permissions Policy specified by the `allow` attribute implements a further restriction on top of the policy specified in the {{HTTPHeader("Permissions-Policy")}} header. It doesn't replace it.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmliframeelement/allow/index.md:13]:
```suggestion
The Permissions Policy specified by the `allow` attribute implements a further restriction on top of the policy specified in the {{HTTPHeader("Permissions-Policy")}} header. It doesn't replace it.

See [`<iframe>`'s Permissions Policy syntax](/en-US/docs/Web/HTTP/Permissions_Policy#iframe_syntax) for more details.
```

File: files/en-us/web/api/htmliframeelement/allow/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+It reflects the `allow` attribute of the {{HTMLElement("iframe")}} element.
+
+## Value
+
+A string indicates the [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy) specified for this {{HTMLElement("iframe")}} element, each policy must be separated by space. See [`<iframe>`'s Permissions Policy syntax](/en-US/docs/Web/HTTP/Permissions_Policy#iframe_syntax) for more details.
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmliframeelement/allow/index.md:19]:
```suggestion
A string indicates the [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy) specified for this {{HTMLElement("iframe")}} element, each policy must be separated by space.
```

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**